cluster/gce/gci/configure-helper.sh

```diff
@@ -1167,6 +1165,9 @@ For Kubernetes copyright and licensing information, see:
 EOF
 }
 
+function override-kubectl {
```

Comment:
I'd prefer if we could edit the user's path rather than use a bindmount.

This bindmount will not survive a reboot, and afaik this isn't re-run on each boot, so this won't persist.

Editing the PATH (preferably with something like Gentoo's `/etc/env.d` if GCI has such an option) seems saner to me.